kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -232,6 +238,24 @@ def account_CHEATCODE_ADDRESS(store_var: KInner) -> KApply:  # noqa: N802
             intToken(0),
         )
 
+    @staticmethod
+    def help_info(kcfg_explore: KCFGExplore, apr_proof: APRProof) -> list[str]:
+        res_lines = []
+        res_lines += print_failure_info(apr_proof.kcfg, apr_proof.id, kcfg_explore)
+
+        print_foundry_success_info = any('foundry_success' in line for line in res_lines)
+        if print_foundry_success_info:
+            res_lines.append('')
+            res_lines.append('See `foundry_success` predicate for more information:')
+            res_lines.append(
+                'https://github.com/runtimeverification/evm-semantics/blob/master/include/kframework/foundry.md#foundry-success-predicate'
+            )
+        res_lines.append('')
+        res_lines.append(
+            'Access documentation for KEVM foundry integration at https://docs.runtimeverification.com/kevm-integration-for-foundry/'
+        )
+        return res_lines
```

Comment:
Let's make this always return only the foundry help lines, not the failure info. The call site then decides whether to invoke `print_failure_info` and `Foundry.help_info`.